domains/atharv.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal website for Atharv",
+  "repo": "https://github.com/realatharv/atharv",
+  "owner": {
+    "username": "realatharv",
+    "email": "atharv.sbiz@gmail.com",
+    "twitter": "realatharv"
+  },
+  "record": {
+    "CNAME": "github-username.github.io"

```

Comment:
404?

Reply:
you need to replace `github-username` with ur gh username.

Reply:
oh wait sorry

Reply:
i changed it

Reply:
404 still exist